huma.go

```diff
@@ -522,340 +522,342 @@ func Register[I, O any](api API, op Operation, handler func(context.Context, *I)
 	a := api.Adapter()
 
 	a.Handle(&op, func(ctx Context) {
-		var input I
-
-		// Get the validation dependencies from the shared pool.
-		deps := validatePool.Get().(*validateDeps)
-		defer func() {
-			deps.pb.Reset()
-			deps.res.Reset()
-			validatePool.Put(deps)
-		}()
-		pb := deps.pb
-		res := deps.res
-
-		errStatus := http.StatusUnprocessableEntity
-
-		v := reflect.ValueOf(&input).Elem()
-		inputParams.Every(v, func(f reflect.Value, p *paramFieldInfo) {
-			var value string
-			switch p.Loc {
-			case "path":
-				value = ctx.Param(p.Name)
-			case "query":
-				value = ctx.Query(p.Name)
-			case "header":
-				value = ctx.Header(p.Name)
-			}
+		api.Middlewares().Handler(HandlerFunc(func(_ API, ctx Context) {
```

Comment:
In handler func, I only add this string, and one more at the end, nothing more :)